src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -26,6 +26,10 @@ $schedule-btn--height: 51px;
     background-color: opaque-black(0.9);
     height: $schedule-btn--height;
     padding-left: $NavBarSchedule--padding;
+
+    &--show {
+      transition: all 400ms $transition-function;
+    }
```

Comment:
I meant a bit different changes.
```suggestion
    transition: all 400ms $transition-function;
    transform: translateX(-40%);

    &--show {
      transform: translateX(0);
    }
```

Reply:
but instead of 40%, can you use a variable, that is already there, please?
`$NavBarSchedule--width`

Reply:
fixed in b29eef631b102c46b08709e69b4377cce1553125